name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/WidgetFactory.java

```diff
@@ -88,7 +88,7 @@
                                     .with(Values.Amount) //
                                     .with((ds, period) -> {
                                         long[] d = data.calculate(ds, period).calculateInvestedCapital();
-                                        return d.length > 0 ? d[d.length - 1] : 0L;
+                                        return d.length > 0 ? d[d.length - 1] - d[0]: 0L;
```

Comment:
The method #calculateInvestedCapital returns an array that cumulates the inbound/outbound transfers (=investments) of the given period of time. Therefore I think the widget does what it says: report the investments for the given reporting period.

Technically this changes nothing, because the first day (index 0) of the array will never have investments because the calculation starts with the valuation at the end of the first day.

I propose the following: can you add an example that shows what is currently displayed and why you expect a different value? Then open the PR again.

Reply:
> Technically this changes nothing, because the first day (index 0) of the array will never have investments because the calculation starts with the valuation at the end of the first day.

This is not correct, because the first day is the first day of the reporting periode and there might be transactions before the reporting periode.

Here is an example where it makes a difference: 1000 € where invested on 2020-08-03:
![image](https://user-images.githubusercontent.com/10776736/128339775-35e9cd56-531b-4220-a619-b1da0370cbc0.png)

Now the invested capital widget shows on 2021-08-05 for the past 1 year (so the reporting periode does not contain the buy transaction) that 1000 € where invested (which is not true, there was no investment in the reporting periode):
![image](https://user-images.githubusercontent.com/10776736/128340253-270928f8-f193-4ffc-b6e1-ce2ef6299b69.png)

With this PR, it shows the correct value (0 €):
![image](https://user-images.githubusercontent.com/10776736/128340456-73fc6f94-748c-467b-9592-8d2c95ac2f01.png)